### PR TITLE
CI fails on error

### PR DESCRIPTION
**Pull request by notdanilo**
### Pull Request Description
Node cmd spawn now exits with error codes if it fails.

### Checklist
Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] All code conforms to the [Rust](https://github.com/luna/enso/blob/master/doc/rust-style-guide.md), [Scala](https://github.com/luna/enso/blob/master/doc/scala-style-guide.md), [Java](https://github.com/luna/enso/blob/master/doc/java-style-guide.md) or [Haskell](https://github.com/luna/enso/blob/master/doc/haskell-style-guide.md) style guides as appropriate.
- [x] All code has been tested where possible.

### CODE REVIEW

File: src/rust/ide/src/controller/module.rs [hunk 1]
```diff
@@ -343,6 +342,8 @@ mod test {
 
     #[wasm_bindgen_test]
     fn update_ast_after_text_change() {
+        //FIXME[DG]: This test is failing because Parser is returning a different IdMap from the one
+        // passed to it. For instance, the second Span(2,1) is returned as Span(2,3).
```
[the remark this answers is not in the document]
Comment (notdanilo) [reply src/rust/ide/src/controller/module.rs]: I just removed this note :)

File: src/rust/lib/data/src/text.rs [hunk 2]
```diff
@@ -79,18 +79,28 @@ impl SubAssign for Size {
 #[derive(Clone,Copy,Debug,Default,PartialEq,Eq,PartialOrd,Ord,Serialize,Deserialize)]
 pub struct Span { pub index:Index, pub size:Size }
 
-impl From<(usize,usize)> for Span {
```
Comment (Josef-Vonasek) [inline src/rust/lib/data/src/text.rs]: Why this change?
